src/Controller/PluginFamily.php

```diff
@@ -482,6 +526,12 @@ public function insert_footer_templates() {
 		if ( ! $this->can_enqueue_admin_assets() ) {
 			return;
 		}
+		// Make notice text available to the included template while preserving default text if empty.
+		$notice_text = $this->notice_text ?? printf(
+			// translators: %1$is = Plugin Name.
+			esc_html__( '%1$s recommends you to optimize your images for even better website performance.', '%domain%' ),
```

Comment:
We can skip escaping html since we already do that in the view [here](https://github.com/pratikrm/plugin-family/blob/90de5acef49562e8ea68c44761f31490ad6550b7/src/View/promote-imagify-uploader.php#L12)